src/python/WMCore/WMRuntime/Bootstrap.py

```diff
@@ -231,12 +237,18 @@ def createInitialReport(job, reportName):
     # $MACHINEFEATURES/total_cpu: number of configured job slots
     # $JOBFEATURES/hs06_job: HS06 score available to your job
     # $JOBFEATURES/allocated_cpu: number of allocated slots (=8 in case of a multicore job
+
+    machineFeaturesFile = os.environ.get('MACHINEFEATURES/hs06')
     report.data.machineFeatures = {}
-    report.data.machineFeatures['HS06_SCORE_HOST'] = os.environ.get('MACHINEFEATURES/hs06')
-    report.data.machineFeatures['TOTAL_CPU'] = os.environ.get('MACHINEFEATURES/total_cpu')
+    if machineFeaturesFile:
+        report.data.machineFeatures['HS06_SCORE_HOST'] = readFloatFromFile("%s/hs06" % machineFeaturesFile)
```

Comment:
How about we name these metrics after their real name from the files?

Reply:
Yes, I think it is better. Thanks Alan for reviewing